File: app/src/main/java/net/harimurti/tv/extra/HttpsTrustManager.java
package net.harimurti.tv.extra;

import android.annotation.SuppressLint;

import java.security.cert.X509Certificate;
import javax.net.ssl.X509TrustManager;

public class HttpsTrustManager implements X509TrustManager {

    private static final X509Certificate[] _AcceptedIssuers = new X509Certificate[] { };

    @SuppressLint("TrustAllX509TrustManager")
    @Override
    public void checkClientTrusted(java.security.cert.X509Certificate[] x509Certificates, String s) { }

    @SuppressLint("TrustAllX509TrustManager")
    @Override
    public void checkServerTrusted(java.security.cert.X509Certificate[] x509Certificates, String s) { }

    public boolean isClientTrusted(X509Certificate[] chain) {
        return true;
    }

    public boolean isServerTrusted(X509Certificate[] chain) {
        return true;
    }

    @Override
    public X509Certificate[] getAcceptedIssuers() {
        return _AcceptedIssuers;
    }

}

File: app/src/main/java/net/harimurti/tv/extra/HttpsTrustManager.kt
package net.harimurti.tv.extra

import android.annotation.SuppressLint
import java.security.cert.X509Certificate
import javax.net.ssl.X509TrustManager

class HttpsTrustManager : X509TrustManager {
    companion object {
        private val AcceptedIssuers = arrayOf<X509Certificate>()
    }

    @SuppressLint("TrustAllX509TrustManager")
    override fun checkClientTrusted(x509Certificates: Array<X509Certificate>, s: String) {
    }

    @SuppressLint("TrustAllX509TrustManager")
    override fun checkServerTrusted(x509Certificates: Array<X509Certificate>, s: String) {
    }

    fun isClientTrusted(chain: Array<X509Certificate?>?): Boolean {
        return true
    }

    fun isServerTrusted(chain: Array<X509Certificate?>?): Boolean {
        return true
    }

    override fun getAcceptedIssuers(): Array<X509Certificate> {
        return AcceptedIssuers
    }
}